datafusion/src/logical_plan/dfschema.rs

```diff
@@ -248,12 +248,14 @@ where
 }
 
 impl ToDFSchema for Schema {
+    #[allow(clippy::wrong_self_convention)]
```

Comment:
maybe it's a better idea to remove https://github.com/apache/arrow-datafusion/pull/581/files#diff-54ef5cfee73dd6cdbd9186bbf5931f4e375ccf002f956d1fe8bf35deafaf56a6R244

Reply:
I don't understand the suggestion